lib/sirius_api/strategies/local_token.rb

```diff
@@ -15,11 +15,14 @@ def store?
 
       def authenticate!
         if access_token.blank?
-          fail 'Missing access token'
-          return
+          return 'Missing access token'
```

Comment:
This doesn't seem right. Bang methods should raise exception when they fail. Plus it’s not very clear to me how you use the message in the return value.